client/src/routes/Data.js

```diff
@@ -3,6 +3,8 @@ import TwoSidedSlider from '../components/TwoSidedSlider';
 import {useEffect, useState} from 'react'
 import Card from '../components/Card';
 import AddCard from '../components/AddCard';
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
I appears that importing `bootstrap/dist/css/bootstrap.min.css` messes with all of the CSS in the client. You might be able to go to that file and fix whatever is causing the CSS styling issue. If you can't pinpoint the problem, you can instead create a new file in the components folder called Navbar.css, set the CSS styling there, and import that instead.